package.json

```diff
@@ -16,10 +16,15 @@
   },
   "dependencies": {
     "@fhir-typescript/r4-core": "0.0.12-beta.11",
+    "@codemirror/lang-javascript": "^6.0.1",
+    "@codemirror/lang-json": "^6.0.0",
     "@mantine/core": "^4.2.4",
+    "@mantine/form": "^4.2.11",
     "@mantine/hooks": "^4.2.4",
     "@mantine/next": "^4.2.4",
     "@mantine/notifications": "^4.2.10",
+    "@monaco-editor/react": "^4.4.5",
```

Comment:
Looks like you installed both the code editor libraries I suggested. Since we went with codemirror, can we remove monaco as a dependency?